components/partials/ActionLinkItem.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <a class="action-link" href="#">
+    <span class="action-link__text">{{ linktext }}</span>
+    <span :class="[iconClass, colorIcon]" class="action-link__icon"></span>
```

Comment:
`colorIcon` creates warning beacuse you do not pass such a prop from parent component